src/shared.js

```diff
@@ -2776,60 +2797,87 @@ export const getCheckedInVisit = (data) => {
 };
 
 export const checkInParticipant = async (data, visitConcept) => {
-    
-    let visits;
-    const user_uid = data.state.uid;
-    let sendBioEmail = false;
+  let visits;
+  const uid = data.state.uid;
+  let shouldSendBioEmail = false;
 
-    if(data['331584571']) {
+  if (data["331584571"]) {
```

Comment:
should we just reference values from Concept ID file?

Reply:
Yes, we should. I just don't know whether I should make changes whenever I see them.
I guess I can do raw CID replacement in `checkInParticipant`.